src/main/java/mocacong/server/controller/CafeController.java

```diff
@@ -92,10 +92,10 @@ public ResponseEntity<CafeReviewUpdateResponse> updateCafeReview(
 
     @Operation(summary = "StudyType별로 카페 조회")
     @SecurityRequirement(name = "JWT")
-    @GetMapping
-    public ResponseEntity<CafeFilterResponse> getCafesByStudyType(@RequestParam(required = false) String studytype,
-                                                                  @RequestBody CafeFilterRequest request) {
-        CafeFilterResponse responseBody = cafeService.filterCafesByStudyType(studytype, request);
+    @PostMapping(value = "/studytype")
+    public ResponseEntity<CafeFilterStudyTypeResponse> getCafesByStudyType(@RequestParam(required = false) String studytype,
```

Comment:
120자를 넘어가는데, 엔터로 한 번 띄워주는 것도 좋을 거 같아요
```suggestion
    public ResponseEntity<CafeFilterStudyTypeResponse> getCafesByStudyType(
    @RequestParam(required = false) String studytype,
```
- 네이버 캠퍼스 핵데이, 우아한테크코스 등의 컨벤션
<img width="653" alt="image" src="https://github.com/mocacong/Mocacong-Backend/assets/57135043/98c1c2bd-7fca-4a57-b38c-fd58b9be88a1">

Reply:
넵 수정했습니다~